index.html

```diff
@@ -212,6 +212,15 @@ <h3 class="dish-name">
         </div>
       </div>
     </section>
+    <section class="fourth-section">
+      <div class="review">
+        <h2 class="review-title">Guest Reviews</h2>
+        <img class="divider" src="resources/images/background/white-divider.png">
+        <p class="review-quote"><span class="quote-symbol">“</span>If you’ve been to one of our restaurants, you’ve seen – and tasted – what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lambda cakes,  muffins, and gourmet coffees make us hard to resist! Stop in today and check us out! <br>
+        <span class="signature">- food inc, New York</span>
```

Comment:
```suggestion
        <div class="signature">- food inc, New York</div>
```
And get this element out to parent.

Reply:
fixed